.github/CONTRIBUTING.md

```diff
@@ -1,66 +1,57 @@
-
-<a name="security"></a>
-#Security issues
-Please do _not_ report security issues here, but email them to security at yoast dot com so we can deal with them securely and quickly.
-
-<a name="contribute"></a>
-#Contribute to Yoast SEO
-
-Community made patches, localisations, bug reports and contributions are very welcome and help make Yoast SEO remains the #1 SEO plugin for WordPress.
+## I want to create a patch
+Community made patches, localizations, bug reports and contributions are very welcome and help Yoast SEO remain the #1 SEO plugin for WordPress.
 
 When contributing please ensure you follow the guidelines below so that we can keep on top of things.
 
-## Getting Started
+#### Getting Started
+If there is no ticket for your issue, submit it first, following the guidelines above.
 
-If there is no ticket for your issue, submit it first, following the above guidelines.
-
-## Making Changes
+#### Making Changes
 
 * Fork the repository on GitHub (make sure to use the trunk branch, not master).
 * Make the changes to your forked repository.
-  * Ensure you stick to the [WordPress Coding Standards](http://make.wordpress.org/core/handbook/coding-standards/) and have properly documented any new functions, actions and filters following the [documentation standards](http://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/).
+* Ensure you stick to the []WordPress Coding Standards](http://make.wordpress.org/core/handbook/coding-standards/) and you properly document any new functions, actions and filters following the [documentation standards](http://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/).
 * When committing, reference your issue (if present) and include a note about the fix.
 * Push the changes to your fork and submit a pull request to the 'trunk' branch of the Yoast SEO repository.
 
-At this point you're waiting on us to merge your pull request. We'll review all pull requests, and make suggestions and changes if necessary.
-
-<a name="patch-welcome"></a>
-## "Patch welcome" issues
+We will review your pull request and merge when everything is in order. We will help you to make sure the code will comply to the standards described above.
 
-Some issues are labeled "Patch-welcome". This means we see the value in the particular enhancement being suggested but have decided for now not to prioritize it. If you however decide to write a patch for it we'll gladly include it after some code review.
+#### 'Patch welcome' issues
+Some issues are labeled 'patch-welcome'. This means we see the value in the particular enhancement being suggested but have decided for now not to prioritize it. If you however decide to write a patch for it, we'll gladly include it after some code review.
 
-# Additional Resources
+#### Additional Resources
```

Comment:
Discussed this and decided to keep this at the bottom as these are resources for the contributors.